etna/pipeline/hierarchical_pipeline.py

```diff
@@ -73,8 +86,10 @@ def raw_forecast(
             Dataset with predictions at the source level
         """
         forecast = super().forecast(prediction_interval=prediction_interval, quantiles=quantiles, n_folds=n_folds)
+        target_columns = tuple(get_target_column_names(columns=forecast.columns))
```

Comment:
In both places you use it you converted results to tuple, does the method have correct output format?

Reply:
Yes, because function designed for general use